test/e2e/storage/csi_mock_volume.go

```diff
@@ -156,6 +156,11 @@ var _ = utils.SIGDescribe("CSI mock volume", func() {
 				destroyCSIDriver(cs, m.config.GetUniqueDriverName())
 			})
 		}
+
+		// Wait for the CSIDriver actually get deployed and CSINode object to be generated.
+		// This indicates the mock CSI driver pod is up and running healthy.
+		err = drivers.WaitForCSIDriverRegistrationOnNode(m.config.ClientNodeSelection.Name, m.config.GetUniqueDriverName(), cs)
+		framework.ExpectNoError(err, "Failed to register CSIDriver %v: %v", m.config.GetUniqueDriverName(), err)
```

Comment:
minor nit: I think `framework.ExpectNoError` will already log `err` and hence there is no need to supply `err` to string format again.

Reply:
Done. I have seen multiple places where we log the err in the explanation part but I guess that should be fine.